app/controllers/users.js

```diff
@@ -0,0 +1,19 @@
+const logger = require('../logger'),
+  users = require('../services/users');
+
+exports.userRegistration = (req, res, next) => {
+  logger.info('POST method start. User registration.');
+  if (req.query) {
```

Comment:
This will always be `True` because express add the query key to the request object, so is pointless. Check for the user key directly.